CSS/styles.css

```diff
@@ -272,21 +272,24 @@ form ul {
   border-top-right-radius: 70px;
   width: 100%;
   height: auto;
+  min-height: 1263px;
   padding: 0 2.4rem;
   margin-top: 11.4rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma design provided. Please reduce the space between your about section and the contact form.  :eyes :eyes: 

![image](https://user-images.githubusercontent.com/98363075/193170263-29155a23-1212-4ac3-9708-9bf2aa19e10a.png)